UI/Web/src/app/statistics/_components/user-stats-info-cards/user-stats-info-cards.component.ts

```diff
@@ -46,4 +46,12 @@ export class UserStatsInfoCardsComponent {
       ref.componentInstance.title = 'Words Read By Year';
     });
   }
+  openTimeSpentReadingByYearList() {
+    const numberPipe = new CompactNumberPipe();
+    this.statsService.getTimeSpentReadingPerYear().subscribe(yearCounts => {
+      const ref = this.modalService.open(GenericListModalComponent, { scrollable: true });
+      ref.componentInstance.items = yearCounts.map(t => `${t.name}: ${numberPipe.transform(t.value)} hours`);
```

Comment:
We have to use localization everywhere, so find or create the appropriate key. 

Same for the title. Usually each component has it's own.